R/system.R

```diff
@@ -1057,7 +1057,7 @@ checkSourceConflict <- function(files){
   for (file in files){
     ret[[file]] <- tryCatch({
       env <- new.env()
-      source(file, local=env)
+      source(file, local=env, encoding = "UTF-8")
```

Comment:
Please add some comment on why it is safe to assume the file is always in UTF-8.